manifests/config.pp

```diff
@@ -0,0 +1,36 @@
+# @private
+#
+# This class is used to configure the jmxtrans config files
+#
+class jmxtrans::config {
+  include ::jmxtrans
+
+  if $::jmxtrans::manage_service_file {
+    case $::service_provider {
+      'systemd': {
+        file { '/usr/lib/systemd/system/jmxtrans.service':
+          ensure  => file,
+          owner   => 'root',
+          group   => 'root',
+          mode    => '0444',
+          content => epp('jmxtrans/jmxtrans.service.epp', { 'user' => $::jmxtrans::user }),
+        }
+        ~>
+        exec { 'jmxtrans systemctl daemon-reload':
+          command     => 'systemctl daemon-reload',
+          refreshonly => true,
+          path        => $::path,
```

Comment:
Huh. I haven't seen this before, and quick googling doesn't show it. Can you point me at docs or where this is defined?